h/static/scripts/session.js

```diff
@@ -59,7 +61,7 @@ function sessionActions(options) {
  *
  * @ngInject
  */
-function session($document, $http, $resource, flash) {
+function session($document, $http, $resource, flash, $rootScope) {
```

Comment:
I try and keep these ridiculous angular parameter lists alphabetised, so `$resource, $rootScope, flash`.